providers/env-var/build.gradle.kts

```diff
@@ -3,9 +3,7 @@ import org.jetbrains.kotlin.gradle.targets.js.testing.KotlinJsTest
 import org.jetbrains.kotlin.gradle.targets.native.tasks.KotlinNativeTest
 
 plugins {
-    alias(libs.plugins.kotlin.multiplatform)
-    alias(libs.plugins.binary.compatibility.validator)
-    alias(libs.plugins.ktlint)
+    id("dev.openfeature.provider-conventions")
```

Comment:
Just to be sure, what is behind `dev.openfeature.provider-conventions` is this referencing something existing outside of the repository?